taxonomies/Message.ttl

```diff
@@ -60,6 +60,16 @@ ids:rejectionReason a owl:ObjectProperty;
     rdfs:label "rejectionReason"@en;
     rdfs:comment "Specifies the reason of the rejection."@en.
 
+# CallBack URL
+# ------------
+
+ids:callbackURL a owl:DatatypeProperty;
+    rdfs:label "callback URL"@en;
+    rdfs:comment "A callback URL must be provided in a request message and a notification message. "@en;
+    rdfs:domain [ owl:unionOf (ids:RequestMessage ids:NotificationMessage )];
```

Comment:
Could we add this callback address to the `ids:Message` in general? Trying to cover asynchronous use cases we have not in mind yet.

Reply:
Hi Julia, 
according to the opinion of @DErCouch 

> From my perspective it is fine if the callback url is part of all request messages and notification messages, response messages don't need to be focused. 

I'd say we don't add callback address to the `ids:Message` in general, because `ids: ResponseMesaages` is one of the subclasses of the `ids:Message`.  

Best,
Chang